src/inference/is.jl

```diff
@@ -1,43 +1,62 @@
 """
-    IS(n_particles::Int)
+    IS()
 
 Importance sampling algorithm.
 
 Note that this method is particle-based, and arrays of variables
 must be stored in a [`TArray`](@ref) object.
 
-Arguments:
-
-- `n_particles` is the number of particles to use.
-
 Usage:
 
 ```julia
-IS(1000)
+IS()
 ```
 
 Example:
 
 ```julia
 # Define a simple Normal model with unknown mean and variance.
 @model gdemo(x) = begin
-    s ~ InverseGamma(2, 3)
-    m ~ Normal(0, sqrt.(s))
+    s ~ InverseGamma(2,3)
```

Comment:
Add space here. Also, it seems changes in https://github.com/TuringLang/Turing.jl/pull/860 is reversed in this PR by strange git errors.